CHANGELOG.md

```diff
@@ -11,18 +11,23 @@ All notable changes to this project will be documented in this file.
 - Run a `containerdebug` process in the background of each Trino container to collect debugging information ([#687]).
 - Support configuring JVM arguments ([#677]).
 - Aggregate emitted Kubernetes events on the CustomResources ([#677]).
+- Support for Trino 470 ([#705]).
 
 ## Changed
 
+- BREAKING: Hard-code `s3.region=us-east-1` until it can be configured by the end user ([#705]).
```

Comment:
> [!CAUTION]
> We either need to merge with this in, or fix the CRD now to handle the s3 region.

Reply:
This was [decided](https://github.com/stackabletech/decisions/issues/46) and implemented in [stackable-operator](https://github.com/stackabletech/operator-rs/pull/959) and [commons-operator](https://github.com/stackabletech/commons-operator/pull/331).